packages/core/src/Actor.ts

```diff
@@ -169,31 +169,35 @@ export class Actor<TEvent extends EventObject, TEmitted> {
     this.current = behavior.initial;
   }
   public start() {
-    this.behavior = this.behavior.receiveSignal(this.context, startSignal);
+    this.current = this.behavior.receive(
+      this.current,
+      startSignal,
+      this.context
+    );
     return this;
   }
   public stop() {
-    this.behavior = this.behavior.receiveSignal(this.context, stopSignal);
+    this.current = this.behavior.receive(
```

Comment:
this should also call `this.mailbox.length = 0` before calling `this.behavior.receive`. `stop()` can be called while flushing and we can't continue processing events after `.stop` has been called

Reply:
Done in 16a398a